setup.py

```diff
@@ -0,0 +1,29 @@
+import os
```

Comment:
please avoid pure `import os` whenever possible and replace it with `import os.path`